controllers/product/getProducts.js

```diff
@@ -0,0 +1,14 @@
+const Product = require('../../models/product')
+
+function getProducts(req, res) {
+  Product.find()
```

Comment:
@aapetsi We agreed async/await, can you use async/await for consistency

Reply:
@Rexben001 will refactor to async/await